resources/dicts/events/death/general.json

```diff
@@ -1152,387 +1109,475 @@
             "any"
         ],
-        "tags": [
+        "sub_type": [
             "old_age"
         ],
+        "tags": [],
         "weight": 20,
         "event_text": "m_c faded more and more as the days went by, until one day, {PRONOUN/m_c/subject} {VERB/m_c/were/was} found dead in {PRONOUN/m_c/poss} nest.",
         "m_c": {
-            "age": [],
-            "status": [],
+            "age": [ "senior adult", "senior" ],
+            "status": [ "warrior", "medicine cat", "mediator", "elder" ],
             "dies": true
         },
         "history": {
+            "cats": "m_c",
             "reg_death": "m_c died of old age."
         }
     },
     {
-        "event_id": "gen_death_",
+        "event_id": "gen_death_age_any4",
+        "biome": ["any"],
         "camp": [
             "any"
         ],
         "season": [
             "any"
         ],
-        "tags": [
+        "sub_type": [
             "old_age"
         ],
+        "tags": [],
         "weight": 20,
         "event_text": "m_c is suddenly struck by a sharp pain in {PRONOUN/m_c/poss} chest. {PRONOUN/m_c/subject/CAP} {VERB/m_c/are/is} found later that day, cold.",
         "m_c": {
-            "age": [],
-            "status": [],
+            "age": [ "senior adult", "senior" ],
+            "status": [ "warrior", "medicine cat", "mediator", "elder" ],
             "dies": true
         },
         "history": {
+            "cats": ["m_c"],
             "reg_death": "m_c died of a heart attack."
         }
     },
     {
-        "event_id": "gen_death_",
+        "event_id": "gen_death_age_any5",
+        "biome": ["any"],
         "camp": [
-            "Any"
+            "any"
         ],
         "season": [
             "any"
         ],
-        "tags": [
-            "old_age",
-            "all_lives"
-        ],
+        "sub_type": [ "old_age" ],
+        "tags": [ "all_lives" ],
         "weight": 20,
         "event_text": "m_c and r_c loved each other so much they could not be parted, even by death. c_n finds their bodies still wrapped around each other in the morning.",
         "m_c": {
-            "age": [],
-            "status": [],
+            "age": [ "senior adult", "senior" ],
+            "status": [ "any" ],
             "relationship_status": [
                 "mates"
             ],
             "dies": true
         },
         "r_c": {
-            "age": [],
-            "status": [],
+            "age": [ "senior adult", "senior" ],
+            "status": [ "any" ],
             "dies": true
-        },
-        "history": {
-            "reg_death": "This cat died from old age life.",
-            "lead_death": "lost all remaining lives from old age."
-        },
-        "relationships": [
-            {
-                "values": [],
-                "amount": 5
-            }
-        ]
+        },
+        "history": {
+            "cats": ["m_c", "r_c"],
+            "reg_death": "This cat died from old age life.",
```

Comment:
"This cat died from old age life."?  seems like "life" is an accidental addition